### PR TITLE
fix: pass all props to transformProps in LineMulti chart

### PR DESCRIPTION
**Pull request by nytai**
💔 Breaking Changes

🏆 Enhancements

📜 Documentation

🐛 Bug Fix
This pr fixes an issue where LineMulti chart doesn't render with the current superset/master code. 

@mistercrunch 
🏠 Internal

### CODE REVIEW

File: packages/superset-ui-legacy-preset-chart-nvd3/src/LineMulti/LineMulti.jsx [hunk 1]
```diff
@@ -31,8 +31,10 @@ const propTypes = {
...
   formData: PropTypes.object,
-  onAddFilter: PropTypes.func,
-  onError: PropTypes.func,
+  hooks: PropTypes.shape({
+    onAddFilter: PropTypes.func,
+    onError: PropTypes.func,
+  }),
 };
 const defaultProps = {
   onAddFilter() {},
```
Comment (kristw) [inline packages/superset-ui-legacy-preset-chart-nvd3/src/LineMulti/LineMulti.jsx]: You don't need this.now.

Comment (nytai) [reply packages/superset-ui-legacy-preset-chart-nvd3/src/LineMulti/LineMulti.jsx]: oh, didn't see that there. Wonder why is was failing on master if it has that fallback. 🤷‍♂

Comment (kristw) [reply packages/superset-ui-legacy-preset-chart-nvd3/src/LineMulti/LineMulti.jsx]: There is an ongoing issue with build tools, probably not related to this.

**Comment by nytai** [comment]
Looks like the build is complaining with a babel error: `Cannot set property inList of [object Object] which has only a getter`

I was getting the same in dev when trying to build.

**Comment by kristw** [comment]
Please rebase

**Review by kristw — APPROVED** [review]

**Comment by mistercrunch** [comment]
🔨 🐛 🔥

**Pull request merged**